Solution1/Solution1.Tests/BL/Services/WeatherServiceTests.cs

```diff
@@ -40,8 +59,69 @@ public async Task GetByCityNameAsync_ReturnedWeatherDTO_Success()
             var result = await _weatherService.GetByCityNameAsync(cityName);
 
             // Assert
-            var expectedWeatherDto = new WeatherDTO() { CityName = cityName, Temp = temp, Comment = "It's fresh." };
+            var expectedWeatherDto = new WeatherDTO() { CityName = cityName, Temp = temp, Comment = comment };
+            Assert.True(new CompareLogic().Compare(expectedWeatherDto, result).AreEqual);
+        }
+
+        [Theory]
+        [InlineData(1)]
+        [InlineData(3)]
+        public async Task GetForecastByCityNameAsync_ReturnedForecastWeatherDTO_Success(int countDays)
+        {
+            // Arrange
+            var startForecast = new DateTime(2022, 10, 12, 00, 00, 00);
+            var weatherPointPeriod = 3;
```

Comment:
you have constant for it

Reply:
or 8 constant, use it, it would be more understandable why you use it - cause in original method you use it as well